protocol/streaming/full_node_streaming_manager.go

```diff
@@ -710,32 +691,6 @@ func getStreamUpdatesForSubaccountUpdates(
 	return streamUpdates, subaccountIds
 }
 
-// SendFinalizedSubaccountUpdates groups subaccount updates by their subaccount ids and
-// sends messages to the subscribers.
-func (sm *FullNodeStreamingManagerImpl) SendFinalizedSubaccountUpdates(
```

Comment:
This method and below interface are unused (as part of internalize [PR](https://github.com/dydxprotocol/v4-chain/pull/2399)) and should've been deleted